packages/catalog-process/src/services/catalogService.ts

```diff
@@ -1184,40 +1197,44 @@ export async function cloneDescriptorLogic({
   }
 
   const descriptor = retrieveDescriptor(descriptorId, eService);
-  const sourceDocument = descriptor.docs[0];
```

Comment:
This was a bug: it was cloning the interface using the first document as data source, and not the interface